_data/meltano/utilities/superset/apache.yml

```diff
@@ -0,0 +1,32 @@
+name: superset
+namespace: superset
+label: Apache Superset
+description: A modern, enterprise-ready business intelligence web application.
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/apache/superset
+variant: apache
+pip_url: apache-superset==1.5.0 markupsafe==2.0.1
+requires:
+  files:
+  - name: files-superset
+    variant: meltano
+settings:
+- name: home
+  value: $MELTANO_PROJECT_ROOT/.meltano/run/superset
```

Comment:
@pnadolny13 Where is this used? I don't see it referenced in https://github.com/meltano/files-superset/pull/12. Is this to set the `SUPERSET_HOME` env var that it reads from?

Note that in the case of Airflow, `AIRFLOW_HOME` is automatically set at runtime, not through a setting, since it shouldn't be overridden: https://github.com/meltano/meltano/blob/main/src/meltano/core/plugin/airflow.py#L19

Reply:
@DouweM yes its tells Superset where to write the system db and any other files. Its not explicitly referenced in the start up script in https://github.com/meltano/files-superset/pull/12 but Superset itself uses it. If I understand correctly, you're saying I should hard code it like `SUPERSET_HOME=$MELTANO_PROJECT_ROOT/.meltano/run/superset` in the startup script rather than allowing users to override in settings? Is that right?